src/features/pages/404/content/not-found.module.scss

```diff
@@ -0,0 +1,23 @@
+@import 'src/features/styles/theme/variables';
+
+.container {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+  padding: 16rem 0;
+}
+.info {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  margin: 16px;
+  text-color: $color-white;
```

Comment:
`text-color` is not a CSS property, you don't need to set it I guess, you can handle it with props.